teflo_init/orchestrate/README.rst

```diff
@@ -0,0 +1,55 @@
+Basic
```

Comment:
What does  Basic mean here , should the heading be Orchestrate_Example ?

Reply:
fixed